src/main/java/com/sahabuddin/blogappapis/controllers/AuthController.java

```diff
@@ -0,0 +1,28 @@
+package com.sahabuddin.blogappapis.controllers;
+
+import com.sahabuddin.blogappapis.payloads.SignInRequest;
+import com.sahabuddin.blogappapis.services.UserService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api")
```

Comment:
Try to use value={path}. For example @RequestMapping(value="/api/auth").
Use meaningful path for better readability.